tests/providers/conftest.py

```diff
@@ -55,3 +59,32 @@ def hook_conn(request):
             )
 
         yield m
+
+
+def mock_hook(hook_class: type[BaseHook], hook_params=None, conn_params=None):
```

Comment:
`conftest.py` it not a good place for store importable stuff, it intends to use for configure tests (some things limited by root conftest), define fixtures and some other stuff which accessible in `pytest` runtime

Reply:
So I just put it in another file then?  Because I've cleaned up a bit the test and made a reusable mock across odbc and jdbc tests so we have less duplicate code

Reply:
I moved it to test_utils under common sql